src/main/java/Message.java

```diff
@@ -35,4 +35,8 @@ public static String emptyInput() {
     public static String enterUserName() {
         return "Please enter your name";
     }
+
+    public static String enterOpportunityStage() {
+        return "Please enter the opportunity stage";
```

Comment:
This would have to be discussed with Becca, but I wonder if `status` or `opportunityStatus` is clearer than `stage`.